x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/translations.tsx

```diff
@@ -234,6 +234,21 @@ export const EQL_SEQUENCE_SUPPRESSION_GROUPBY_VALIDATION_TEXT = i18n.translate(
   }
 );
 
+export const MACHINE_LEARNING_SUPPRESSION_DISABLED_LABEL = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.machineLearningSuppressionDisabledLabel',
+  {
+    defaultMessage: 'Machine Learning jobs must be running to enable alert suppression.',
+  }
+);
+
+export const MACHINE_LEARNING_SUPPRESSION_INCOMPLETE_LABEL = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.machineLearningSuppressionIncompleteLabel',
+  {
+    defaultMessage:
+      'This list of fields may be incomplete as some Machine Learning jobs are not running.',
```

Comment:
What should users do if they notice that certain fields are missing? We should prob include that suggestion so they have a path forward from here.

Reply:
We could provide the list of non-running ML jobs, but that could get verbose. As with the above, the solution is to start ML jobs.

Reply:
Makes sense. I also wonder how useful that list would be for users who don't know which jobs to enable. 

How's this:

```suggestion
      'This list of fields might be incomplete as some Machine Learning jobs are not running. Start more jobs to add more fields.',
```